src/com/company/exercises/FIs.java

```diff
@@ -0,0 +1,46 @@
+package com.company.exercises;
+
+import java.util.function.Consumer;
+import java.util.function.Function;
+import java.util.function.Predicate;
+import java.util.function.Supplier;
+
+public class FIs {
```

Comment:
I know these classes are just meant to keep you exercises separate, but avoid using abbreviations for class names (or variables for that matter) if at all possible. Only use them if the item in question is extremely common, like DTO etc.

Reply:
Done.